scripts/install.sh

```diff
@@ -10,6 +10,8 @@ initPlatform() {
         PLATFORM="darwin_amd64"
     elif [ "$OS" = "linux" ] && [ "$ARCH" = "x86_64" ]; then
         PLATFORM="linux_amd64"
+    elif [ "$OS" = "linux" ] && [ "$ARCH" = "arm64" ]; then
```

Comment:
I think on a lot of modern arm64 linux system `ARCH` will resolve to `aarch64` not `arm64`, so we might need to add support for both cases here

Reply:
Good catch! Just updated the code to:
```bash
    elif [ "$OS" = "linux" ] && { [ "$ARCH" = "arm64" ] || [ "$ARCH" = "aarch64" ]; }; then
        PLATFORM="linux_arm64"
```